leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/LeshanServerBuilder.java

```diff
@@ -269,6 +274,15 @@ public LeshanServerBuilder setDtlsConfig(DtlsConnectorConfig.Builder config) {
     }
 
     /**
+     * Used to create custom CoAP endpoint, this is only for advanced users. <br>
+     * DTLSConnector is expected for secured endpoint.
```

Comment:
We should explain here that localAddress/dtlsConfig are ignored if an endpoint factory is provided.